scripts/zones/The_Colosseum/npcs/Zandjarl.lua

```diff
@@ -0,0 +1,60 @@
+-----------------------------------
+--  Area: The Colosseum
+--  NPC:  Zandjarl
+--  Type: Pankration NPC
+-----------------------------------
+package.loaded["scripts/zones/The_Colosseum/TextIDs"] = nil;
```

Comment:
You aren't using the code I gaaaave yooooouuuuu....
Multiple things wrong here.